### PR TITLE
1041 paginate people modal

### PR DESCRIPTION
**Pull request by EDsCODE**
## Changes
- addresses #1041 
-

## Checklist
- [x] All querysets/queries filter by Team (if applicable)
- [x] Backend tests (if applicable)
- [ ] Cypress E2E tests (if applicable)

### CODE REVIEW

File: posthog/api/action.py [hunk 1]
```diff
@@ -210,8 +211,8 @@ def _calculate_people(events: QuerySet):
                     return Response([])
                 filtered_events = process_entity_for_events(entity, team_id=team.pk, order_by=None).filter(filter_events(team.pk, filter, entity))
 
-        people = _calculate_people(events=filtered_events)
-        return Response([people])
+        people = _calculate_people(events=filtered_events, offset=offset)
+        return Response({'result': [people], 'offset': offset + 100})
```
Comment (timgl) [inline posthog/api/action.py:215]: It'd be good to make this API consistent with the other APIs in terms of pagination: https://posthog.com/docs/api/api#pagination

**Review by timgl — COMMENTED** [review]
I can't seem to make this work. The API loads correctly but it gets stuck loading
image

File: posthog/api/action.py (hunk 1 above)
Comment (EDsCODE) [reply posthog/api/action.py]: done!

**Comment by EDsCODE** [comment]
not sure why it was inifinite loading for you. i just updated the pagination format so try it again and lmk if it still infinite loads!

**Comment by EDsCODE** [comment]
linted the test_action page so diffs are huge

**Pull request merged**